### PR TITLE
Indicators

### LINKED ISSUE #28: Calculate indicators

### PR DESCRIPTION
**Pull request by rodrigommorais**
Close #28

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
 from tatu_re.utils import get_data
+from tatu_re.indicators import calculate_indicators
 
 #------------- S&P Data
 start = datetime(2020,3,1)
 end = datetime(2020,3,30)
 df = get_data(start, end)
 
+#------------- Indicators calculation
+indicators = calculate_indicators(start)
```
Comment (aateg) [inline main.py:14]: @rodrigommorais isso aqui ta desatualizado ne?

**Comment by aateg** [comment]
TA-lib nao funcionou pra mim
vc instalou com pip install TA-lib?

**Comment by rodrigommorais** [comment]
Procurando no google "https://github.com/mrjbq7/ta-lib"

**Comment by aateg** [comment]
to test this PR:
```python
from datetime import datetime
from tatu_re.utils import get_data
from tatu_re.indicators import calculate_indicators

start = datetime(2019,3,30)
end = datetime(2020,3,30)
df = get_data(start, end)

ind = calculate_indicators(df)
print(ind.head())
```

**Review by guilhermefgs — APPROVED** [review]

**Pull request merged**